app/(dashboard)/settings/components/DeleteAccount.tsx

```
@@ -0,0 +1,126 @@
+  }
+
+  const deleteAccountMutation = useMutation(
+    () => {
+      return deleteUser();
+    },
+    {
+      onSuccess: () => {
+        snackbar.showSuccess("Your account has been successfully deleted");
+        handleLogout();
+      },
+      onError: (error) => {},
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)